src/main/java/com/jcabi/github/mock/MkReleases.java

```diff
@@ -176,4 +205,37 @@ private String xpath() {
             this.coords
         );
     }
+
+    /**
+     * Check the first release is greater than last release.
+     * @param first First release.
+     * @param last Last release.
+     * @return True/False.
+     */
+    public boolean createLatest(
+            final Release first,
+            final Release last
+    ) {
+        return this.createdAt(first)
+                .compareTo(this.createdAt(last)) > 0;
+    }
+
+    /**
+     * Veriffy the date time release was created.
+     * @param release Release date.
+     * @return Zero or DateTime.
+     */
+    private Long createdAt(
+            final Release release
+    ) {
+        try {
+            final Release.Smart smart = new Release.Smart(release);
+            final Date created = smart.createdAt();
+            if (created != null) {
+                return created.getTime();
+            }
+        } catch (final IOException iox) {
```

Comment:
@weisebrazil please don't swallow exception, you can wrap it inside `IllegalStateException`, and throw that one.